homeassistant/components/thermostat/zwave.py

```diff
@@ -116,6 +116,8 @@ def update_properties(self):
             temps.append(int(value.data))
             if value.index == self._index:
                 self._target_temperature = value.data
+                if self._unit is None:
```

Comment:
Not frontend :P component. :+1: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/climate/__init__.py#L361